### ISSUE TITLE
attrs plugin doesn't recognize new eq and order options

### ISSUE THREAD

**Issue by belm0** [python/mypy#7663]
attrs 19.2.0 added `eq` and `order` options, replacing `cmp`

```python
from attr import attrs

@attrs(eq=False)
class Foo:
    def __eq__(self, other):
        pass
```

Actual:
```
foo.py:3: error: Name '__eq__' already defined on line 2
```

Expected:  no errors

mypy 0.720
attrs 19.2.0

**Comment by euresti** [comment]
Try the latest master.

**Comment by JelleZijlstra** [comment]
Fixed on master
